extensions/resteasy-classic/resteasy-client/runtime/src/main/java/io/quarkus/restclient/runtime/QuarkusRestClientBuilder.java

```diff
@@ -148,6 +165,13 @@ public RestClientBuilder queryParamStyle(QueryParamStyle queryParamStyle) {
         return this;
     }
 
+    @Override
+    public RestClientBuilder header(final String name, final Object value) {
```

Comment:
So this yet another way for users to be able to set headers?

Reply:
Yes, they added that in the builder API.